cmd/fleetctl/get_test.go

```diff
@@ -509,6 +511,8 @@ spec:
   host_settings:
     enable_host_users: true
     enable_software_inventory: false
+  integrations:
+    jira: null
```

Comment:
Double check, will this break old/previous versions of `fleetctl` running against new version of `fleet`?

Reply:
Is that something we support? That seems tricky in the long run - e.g. with API versioning we will want to use `/api/latest` in fleetctl, but by it's very nature this means getting new endpoints that may have breaking changes (e.g. different response payloads, or paths that don't exist anymore).

In this specific case, I know that the PATCH config endpoint fails if there are unknown keys, but this would be the other way around (old fleet, newer fleetctl).